keps/sig-testing/2291-presubmit-config-inside-the-tested-repo /kep.yaml

```diff
@@ -0,0 +1,17 @@
+title: Presubmit config inside the tested repo
```

Comment:
nit: there is a space at the end of the folder name. Probably that should be removed?

Reply:
Thanks for the review. I updated it.